portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -1,10 +1,17 @@
 package com.google.sps.servlets;
 
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+//import org.jsoup.Jsoup;
```

Comment:
If these are unnecessary dependencies, lets remove them rather than comment them out

Reply:
I'll make sure to follow new instructions and respond to comments before merging.

Thanks for the advice! I already uncommented it, I used it later in another commit.